src/main/kotlin/com/sourcegraph/common/NotificationGroups.kt

```diff
@@ -0,0 +1,9 @@
+package com.sourcegraph.common
+
+object NotificationGroups {
```

Comment:
We already have a message bundle, why there we want to use separate object?

Reply:
these are not messages - those are notification group IDs